### ISSUE TITLE
The xcat-dep.repo in xcat-dep online repo is not correctly 

### ISSUE THREAD

**Issue by hu-weihua** [xcat2/xcat-core#6339]
The URL path of ``xcat-dep.repo`` file under ``https://xcat.org/files/xcat/repos/yum/devel/xcat-dep/rh7/x86_64/`` is not correct. Miss ``devel`` part.

https://xcat.org/files/xcat/repos/yum/devel/xcat-dep/rh7/x86_64/xcat-dep.repo
```
[xcat-dep]
name=xCAT 2 depedencies
baseurl=http://xcat.org/files/xcat/repos/yum/xcat-dep/rh7/x86_64
enabled=1
gpgcheck=1
gpgkey=http://xcat.org/files/xcat/repos/yum/xcat-dep/rh7/x86_64/repodata/repomd.xml.key
```

**Comment by whowutwut** [comment]
Wow, this is kind of unfortunate, it looks like all the devel `xcat-dep.repo` files are wrong.  This is because we are building xcat-dep and just putting a version of it under the `devel` subdirectory....

On the server...  none of them have devel... 
```
$ pwd
/path......./files/xcat/repos/yum/devel/xcat-dep
$ grep -R baseurl * | grep xcat-dep.repo
rh7/s390x/xcat-dep.repo:baseurl=http://xcat.org/files/xcat/repos/yum/xcat-dep/rh7/s390x
rh7/ppc64le/xcat-dep.repo:baseurl=http://xcat.org/files/xcat/repos/yum/xcat-dep/rh7/ppc64le
rh7/x86_64/xcat-dep.repo:baseurl=http://xcat.org/files/xcat/repos/yum/xcat-dep/rh7/x86_64
rh7/ppc64/xcat-dep.repo:baseurl=http://xcat.org/files/xcat/repos/yum/xcat-dep/rh7/ppc64
sles12/ppc64le/xcat-dep.repo:baseurl=http://xcat.org/files/xcat/repos/yum/xcat-dep/sles12/ppc64le
sles12/x86_64/xcat-dep.repo:baseurl=http://xcat.org/files/xcat/repos/yum/xcat-dep/sles12/x86_64

```

Not  sure yet  how to fix this.

**Comment by gurevichmark** [comment]
Do repo files from `xcat.org` actually get used at runtime ?
When I run `go-xcat --xcat-version=devel install` the local `/etc/yum/yum.repos.d/xcat-dep.repo` contains correct info:
```
[root@briggs01 yum.repos.d]# cat xcat-dep.repo
[xcat-dep]
name=xcat-dep
baseurl=http://xcat.org/files/xcat/repos/yum/devel/xcat-dep/rh8/ppc64le
enabled=1
gpgcheck=1
gpgkey=http://xcat.org/files/xcat/repos/yum/devel/xcat-dep/rh8/ppc64le/repodata/repomd.xml.key
[root@briggs01 yum.repos.d]#
```

**Comment by whowutwut** [comment]
Similar to #3322

**Comment by gurevichmark** [comment]
Fixed by #6444
